examples/plot_raster_stack_crop.py

```diff
@@ -0,0 +1,135 @@
+"""
+Stack and Crop Rasters with EarthPy
+====================================
+
+Learn how to stack and crop satellite imagery with GeoPandas objects in EarthPy
+"""
+
+###############################################################################
+# Stacking and Cropping Rasters using EarthPy
+# ---------------------------------------------
+#
+# .. note:: The examples below will show you how to use the ``es.stack()`` and ``es.crop_image()`` functions.
+
+
+###############################################################################
+# Stacking Multi Band Imagery
+# -----------------------------
+#
+# Let's explore a simple raster stack using EarthPy. To begin, import the needed packages
+# and create an array to be plotted. Below we plot the data as continuous with a colorbar
+# using the ``plot_bands()`` function.
+
+###############################################################################
+# Import Packages
+# ------------------------------
+#
+# To begin, import the needed packages. You will primarily be using the spatial module
+# of EarthPy, but other packages will be needed as well.
+
+import os
+from glob import glob
+import matplotlib.pyplot as plt
+import rasterio as rio
+from rasterio.plot import plotting_extent
+import earthpy as et
+import geopandas as gpd
+import earthpy.spatial as es
+import earthpy.plot as ep
+
+
+###############################################################################
+# Get Example Data Ready for Stack
+# ----------------------------------
+# Create a stack from all of the Landsat .tif files (one per band) with the
+# `es.stack()` function. Make sure you output a raster with the `out_path`
+# argument, as it will be needed for the crop.
+
+# Get data and set your home working directory
+
+data = et.data.get_data("cs-test-landsat")
+data2 = et.data.get_data("cold-springs-fire")
+os.chdir(os.path.join(et.io.HOME, "earth-analytics"))
+os.environ[
```

Comment:
@nkorinek i know that this is needed for things to run on your compute but please remove it for this PR as it's specific to your system. what i suggest you do instead is create a box as a `notes` that explains that on some windows system, a user might need to add a line similar to this (but don't use your username)
Here is how you add a breakout box in a vignette: 

```
# .. note:: The examples below will show you how to use the ``draw_legend()`` function for creating plots
```